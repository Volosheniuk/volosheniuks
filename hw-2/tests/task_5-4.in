i9 8 7 5 6 4 3
1.0 2.23 4.68 4.52 9.31 2.1 2.0
