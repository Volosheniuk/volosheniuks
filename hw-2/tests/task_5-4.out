42
25.84
